calico_kubernetes/calico_kubernetes.py

```diff
@@ -710,10 +710,11 @@ def _label_to_tag(self, label_key, label_value):
         return tag
 
 
-if __name__ == '__main__':
-    configure_logger(logger, LOG_LEVEL, True)
-    configure_logger(pycalico_logger, LOG_LEVEL, False)
-
+def main():
+    """
+    Runs the plugin.
+    """
+    logger.info('Plugin Args: %s' % sys.argv)
```

Comment:
Yep, good catch.  That seems to be the only one of the logs that was done that way....